adr/ADR-6.md

```diff
@@ -53,4 +56,6 @@ durable-name       = term
 js-internal-prefix = dollar (prefix dot)+
 js-user-prefix     = (prefix dot)+
 account-name       = (filename-safe)+ maximum 255 characters
+kv-bucket-name     = restricted-term
+kv-key-name        = term
```

Comment:
key name is not `term` at all.

`.` is allowed but not at start, all the various punctuations in 33-126 ascii is not allowed etc. It's `[-/_=\.a-zA-Z0-9]` and may not start with `.`

Reply:
okay was not sure, figure you would catch that in review. I will update.